code-challenges.test.js

```diff
@@ -17,38 +20,93 @@
 // --------------------1) Create a function that takes in a number (greater than 2) and returns an array of that length containing the numbers of the Fibonacci.
 
 // a) Create a test with expect statements for each of the variables provided.
+test('when nums is called, you should get an array containing the fibonacci numbers for number that you input', () => {
+    let fibNumbers = [1, 1, 2, 3, 5, 8]
+    expect(nums).toEqual(fibNumbers)
```

Comment:
Looks like these tests aren't actually calling a `nums` function because there hasn't been one declared rather it's calling on a variable named `nums`. The logic you have below looks like it should be encapsulated by declaring a function.